bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/ConfigTomlParser.java

```diff
@@ -113,7 +116,9 @@ private static TomlTableNode retrieveModuleNode(TomlTableNode tomlNode, Module m
         String orgName = module.getOrg();
         String moduleName = module.getName();
         if (tomlNode.entries().containsKey(orgName)) {
-            tomlNode = (TomlTableNode) tomlNode.entries().get(orgName);
+            TomlNode orgNode = tomlNode.entries().get(orgName);
+            checkModuleStructure(orgNode, orgName + "." + moduleName);
```

Comment:
Shall we pass the orgNode and moduleName to a method and do the concatenation inside the method.